jumpscale/entry_points/usershell.py

```diff
@@ -32,10 +37,36 @@
 )
 
 
+def get_binary_link():
+    resp = requests.get("https://api.github.com/repos/threefoldtech/js-ng/releases/latest")
+    resp = resp.json()
+    # get versions
+    download_link = ""
+    version = resp["tag_name"]
+    for platform in resp["assets"]:
+        if sys.platform in platform["name"]:
+            download_link = platform["browser_download_url"]
+    return version, download_link
+
+
+def update():
+    print("checking for updates")
+    latest_version, binary_link = get_binary_link()
+    current_version = pkg_resources.get_distribution("js-ng").version
```

Comment:
could we abstract this line somewhere in js-ng itself so we don't repeat this pattern? Just incase we change the way we version stuff in the future